sklearn/utils/__init__.py

```diff
@@ -139,6 +139,16 @@ def safe_mask(X, mask):
     -------
     mask : ndarray
         Array that is safe to use on X.
+
+    Examples
+    --------
+    >>> import numpy as np
+    >>> from sklearn.utils import safe_mask
+    >>> data = np.array([1, 2, 3, 4, 5])
+    >>> condition = [False, True, True, False, True]
+    >>> mask = safe_mask(data, condition)
+    >>> data[mask]
+    array([2, 3, 5])
```

Comment:
Please also apply it to a CSR sparse array with 5 rows, 1 col and display the the result of calling `.toarray()` on the returned value.